hnn_core/gui/gui.py

```diff
@@ -1000,8 +1051,18 @@ def add_drive_widget(drive_type, drive_boxes, drive_widgets, drives_out,
                 default_delays=prespecified_delays,
             )
 
+        # notice this is a special case.
+        elif drive_type == 'Tonic':
+            name = drive_type + str(len(drive_boxes))
+            drive, drive_box = _get_tonic_widget(
+                name,
+                layout,
+                style,
+                data=prespecified_drive_data
+            )
+
         if drive_type in [
-                'Evoked', 'Poisson', 'Rhythmic', 'Bursty', 'Gaussian'
+                'Evoked', 'Poisson', 'Rhythmic', 'Bursty', 'Gaussian', 'Tonic'
```

Comment:
Probably not related to this PR, but do we still need references to the `'Gaussian'` drive type anywhere in the GUI?

Reply:
yes here

https://github.com/chenghuzi/hnn-core/blob/4cf3dc3b7b33fff2c96280fb78b0bbfe652eabd7/hnn_core/gui/gui.py#L1042

Reply:
humm ... but I don't get how you can have a Gaussian drive type. It's not an option in the GUI